ql/src/test/queries/clientpositive/auto_sortmerge_join_17.q

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE tbl1_n5(key int, value string) CLUSTERED BY (key) SORTED BY (key) INTO 2 BUCKETS;
+
+insert into tbl1_n5(key, value)
+values
+(0, 'val_0'),
+(2, 'val_2'),
+(9, 'val_9');
+
+set hive.optimize.semijoin.conversion = false;
```

Comment:
It wasn't clear why  this config has to be set false for this test case.  If it is needed, can you add a comment in the test file ?

Reply:
It is not necessary for the repro. I kept it because I wanted to see of the issue can be reproduced with inner join too.
Removed from the final version.